queries/ocaml/highlights.scm

```diff
@@ -71,7 +71,7 @@
 ; Constants
 ;----------
 
-[(boolean) (unit)] @constant
+[(boolean) (unit)] @constant.builtin
```

Comment:
This looks good but boolean should be `@boolean`

Reply:
done